en/technical-reference/docs-changelog.md

```diff
@@ -3,6 +3,9 @@
 
 <!-- #### Unreleased 
 -->
+#### 2022/08/02
+- update Snap Javascript Callback and add link to Snap Docs.
+
```

Comment:
> update Snap Javascript Callback

That's ambiguous, you are not updating the callback. You are just clarifying the docs. Should be:
> improve Snap JS callback advanced section by adding link to Snap Docs.